packages/admin/src/utils/general.ts

```diff
@@ -24,10 +24,16 @@ const getObjEntriesByKeySubstr = (substr: string, obj: object) => {
   return Object.entries(obj).filter(([key]) => key.includes(substr));
 };
 
-const getLoginUrl = () => {
-  return process.env.ONE_LOGIN_ENABLED === 'true'
-    ? process.env.V2_LOGIN_URL
-    : process.env.LOGIN_URL;
+type GetLoginUrlOptions = {
+  redirectToApplicant?: boolean;
+};
+
+const getLoginUrl = (options?: GetLoginUrlOptions) => {
+  const oneLoginEnabled = process.env.ONE_LOGIN_ENABLED === 'true';
+  if (options?.redirectToApplicant && oneLoginEnabled) {
+    return `${process.env.USER_SERVICE_URL}/v2/login?redirectUrl=${process.env.APPLICANT_DOMAIN}`;
```

Comment:
prob wanna set redirect to `isNewApplicant` juuust in case?

```suggestion
    return `${process.env.V2_LOGIN_URL}?redirectUrl=${process.env.APPLICANT_DOMAIN}/api/isNewApplicant`;
```

Reply:
Good shout re `/api/isNewApplicant` -`V2_LOGIN_URL` comes with a redirectUrl param already so will need to keep using USER_SERVICE_URL